src/TLuaInterpreter.cpp

```diff
@@ -5640,7 +5640,7 @@ void TLuaInterpreter::initLuaGlobals()
         mpHost->postMessage(modLoadMessageQueue.dequeue());
     }
 
-    loadLuaModule(modLoadMessageQueue, QLatin1String("rex_pcre"), tr("Some functions may not be available."));
+    loadLuaModule(modLoadMessageQueue, QLatin1String("rex_pcre2"), tr("Some functions may not be available."));
```

Comment:
I was thinking more along the lines of trying both rocks - in case the user still has the old one around (like I have) - in either case the library would appear in the `rexlib_pcre` namespace  (so existing references to that in old scripts/packages will continue to work) - though we'd still need a wrapper to provide `fullinfo()` from `patterninfo()` if the former function is not seen in the library that ***is*** loaded...
```suggestion
    loaded = loadLuaModule(modLoadMessageQueue, QLatin1String("rex_pcre"));
    if (!loaded) {
         loadLuaModule(modLoadMessageQueue, QLatin1String("rex_pcre2"), tr("Some regular expression functions may not be available."), QString(), qsl("rex_pcre"));
    }

```